sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/linkedin/impl/OperationsLogger.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.benchmark.linkedin.impl;
+
+import com.azure.cosmos.benchmark.linkedin.impl.models.CollectionKey;
+import com.google.common.base.Preconditions;
+import java.time.Duration;
+import java.util.Objects;
+import java.util.Optional;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Logger class for logging operation specific information
+ */
+public class OperationsLogger {
+    private static final Logger LOGGER = LoggerFactory.getLogger(OperationsLogger.class);
+
+    private final Duration _minLatencyForPrintingDebugInfo;
+
+    public OperationsLogger(final Duration minLatencyForPrintingDebugInfo) {
```

Comment:
I see we have 100 millisec . During cross region (failover)communication almost all the the calls takes more than this in out current sdk ctl , this will blow up the logs in Kusto, also slow down the throughput on client side as we it take some computation to compute request diagnostics

Reply:
Fair point, I'll set it to a very high value. This code could be deleted, but it's something we had on our side. LMK what you prefer.